integration_tests/test_dbapi.py

```diff
@@ -371,3 +371,14 @@ def test_invalid_query_throws_correct_error(trino_connection):
             """,
             params=(3,),
         )
+
+
+def test_eager_loading_cursor_description(trino_connection):
+    cur = trino_connection.cursor()
+    cur.execute('SELECT * FROM system.runtime.nodes')
+    des_before = cur.description
```

Comment:
nit: avoid abbreviations. `before` and `after` are good alternatives (and still clear).